apps/docs/src/app/platform/component-docs/platform-forms/switch/switch-header/switch-header.component.html

```diff
@@ -0,0 +1,8 @@
+<header>Switch</header>
+<description>
+    <p>Switch description</p>
+</description>
+<import module="SwitchModule"></import>
```

Comment:
Missing actual description what we usually add to API docs:

Checkout other components in the platform where we not try to say the same what you can read in core but at minimum how we are enhancing this.

Reply:
fixed